Makefile

```diff
@@ -5,6 +5,13 @@ ORG=timescaledev
 PG_VER=pg16
 PG_VER_NUMBER=$(shell echo $(PG_VER) | cut -c3-)
 PG_MAJOR_VERSION=$(shell echo $(PG_VER_NUMBER) | cut -d. -f1)
+ifeq ($(shell test $(PG_MAJOR_VERSION) -ge 17; echo $$?),0)
+  ALPINE_VERSION=3.21
+  CLANG_VERSION=19
```

Comment:
should we always do latest here at least for pg17?

Reply:
I think the clang version is fixed by the alpine one and I want to fix the OS version I think